src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -87,10 +116,18 @@ internal class AuthorityInfo
 
         public string Host { get; }
         public string CanonicalAuthority { get; }
-        public AuthorityType AuthorityType { get; }
+        internal AuthorityType AuthorityType { get; }
```

Comment:
Shouldn't matter since enclosing AuthorityInfo is internal?

Reply:
Just getting closer to O.